src/ripple/app/paths/impl/Steps.h

```diff
@@ -250,6 +254,7 @@ toStrand (
    @param dst Account that is receiving assets
    @param deliver Asset the dst account will receive
                   (if issuer of deliver == dst, then accept any issuer)
+   @param limitQuality Do not use liquidity below this quality threshold
```

Comment:
That's only true for offers when offer crossing, it's not true for payments or direct steps. We probably want to rename this parameter.